CommunityFridgeMapApi/tests/unit/test_tag.py

```diff
@@ -0,0 +1,53 @@
+import unittest
+from dependencies.python.db import Tag
+
+class DynamoDbMockPutItem():
+    def __init__(self):
+        pass
+
+    def put_item(self, TableName=None, Item=None, ConditionExpression=None):
+        pass
+
+class TagTest(unittest.TestCase):
+
+    def test_is_valid_tag_name(self):
```

Comment:
Can we refactor this test slightly for easier modification and readability? Something like
```
tag = Tag(…)
expected_false = [tag.is_valid_tag_name('')[0], tag.is_valid_tag_name(None)[0], …]
expected_equal = [(tag.is_valid_tag_name('')[1], 'tag_name is an empty string'), (tag.is_valid_tag_name(None)[1], 'tag_name is None'), …]
for condition in expected_false:
    self.assertFalse(condition)
for condition in expected_equal:
    self.assertEqual(condition)
```